memory/src/main/scala/filodb.memory/format/vectors/LongBinaryVector.scala

```diff
@@ -138,6 +138,7 @@ trait LongVectorDataReader extends VectorDataReader {
    */
   def binarySearch(vector: BinaryVectorPtr, item: Long): Int
 
+  def changes(vector: BinaryVectorPtr, start: Int, end: Int): Double
```

Comment:
I would change the signature of `changes` method to return Int.  There's really no reason to return Double here, and it potentially introduces other problems.

Reply:
Changed it to int